### PR TITLE
webgui: Restructure for clarity

### PR DESCRIPTION
**Pull request by pilino1234**
Move all images into images/ directory and update related files

There were two different ajax-loader.gif images, but only one was used. The other one is deleted now.

### CODE REVIEW

**Comment by pilino1234** [comment]
Note: still needs testing.

File: libs/webgui/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
 <meta http-equiv="content-type" content="text/html; charset=UTF-8">
 <title>pilight webgui</title>
-<link rel="icon" type="image/x-icon" href="favicon.ico">
+<link rel="icon" type="image/x-icon" href="iamges/favicon.ico">
```
Comment (CurlyMoo) [inline libs/webgui/index.html]: This doesn't seem right.

Comment (pilino1234) [inline libs/webgui/index.html]: It isn't 😉

**Comment by pilino1234** [comment]
Rebased

**Comment by CurlyMoo** [comment]
Can you integrate this in the webserver unit test as well?

**Comment by pilino1234** [comment]
Done, the webserver unittests pass

(arping is crashing hard, most likely unrelated though) 😕

**Comment by CurlyMoo** [comment]
Do the tests run locally?

**Comment by CurlyMoo** [comment]
You need to check various file sizes:
```
1) test_webserver_get: /home/pilight/source/temp/tests/webserver.c:153: expected <6065> but was <6261>
...
1) test_webserver_get: /home/pilight/source/temp/tests/webserver.c:153: expected <4782> but was <6242>
...
1) test_webserver_get: /home/pilight/source/temp/tests/webserver.c:153: expected <1639> but was <1674>
```

**Comment by pilino1234** [comment]
Should be fixed now, although it would be good if someone else can confirm. I'm back on my old dev computer though (where random unittests fail with coredumps etc.), so I don't trust it and will let travis run the test for me 😄

**Comment by pilino1234** [comment]
The tests run fine on travis now: 

```
[ test_webserver_get                               ]
[ - http://../                                     ]
[ - http://../jquery-2.1.4.min.js                  ]
[ - http://../jquery.mobile-1.4.5.min.js           ]
[ - http://../moment.min.js                        ]
[ - http://../pilight.js                           ]
[ - http://../jquery.mobile-1.4.5.min.css          ]
[ - http://../pilight.css                          ]
[ - http://../pilight.jquery.theme.css             ]
[ - http://../images/ajax-loader.gif               ]
[ - http://../config                               ]
[ - https://../                                    ]
[ - https://../jquery-2.1.4.min.js                 ]
[ - https://../jquery.mobile-1.4.5.min.js          ]
[ - https://../moment.min.js                       ]
[ - https://../pilight.js                          ]
[ - https://../jquery.mobile-1.4.5.min.css         ]
[ - https://../pilight.css                         ]
[ - https://../pilight.jquery.theme.css            ]
[ - https://../images/ajax-loader.gif              ]
[ - https://../config                              ]
[ - https://../nonexisting                         ]
[ test_webserver_whitelist                         ]
[ - http://../                                     ]
[ test_webserver_auth                              ]
[ - http://test:test@../                           ]
[ - http://test1:test1@../                         ]
[ test_webserver_websocket1                        ]
[ test_webserver_websocket2                        ]
.....

OK (5 tests)
```

Will remove wip commits and squash

**Comment by pilino1234** [comment]
(The proc unittest still randomly fails: https://travis-ci.org/pilino1234/pilight/builds/271334190#L1227)

**Comment by CurlyMoo** [comment]
Just restart the test for one or two times and if it keeps failing something is going on.

**Pull request merged**